### PR TITLE
Get rid of legacy

### PR DESCRIPTION
**Pull request by ghost**
https://github.com/tonlabs/TON-Surf/pull/2894

### CODE REVIEW

**Review by SAnatoliiS — APPROVED** [review]

**Review by sertony — APPROVED** [review]

File: packages/components/src/UICompatibilityView/index.web.js [hunk 1]
```diff
@@ -1,17 +0,0 @@
-// @flow
-import React from 'react';
-import { Platform, StyleSheet, Text, View } from 'react-native';
-// TODO: we have `mobile-detect` in UIDevice
-// maybe use only one lib for
-import Bowser from 'bowser';
```
Comment (sertony) [inline packages/components/src/UICompatibilityView/index.web.js:6]: Btw, we can remove the `bowser` from dependencies then

File: packages/components/src/UICompatibilityView/index.web.js [hunk 2]
```diff
@@ -1,17 +0,0 @@
...
-
-import UICompatibilityViewCommon from './UICompatibilityViewCommon';
-
-export default class UICompatibilityView extends React.Component<*> {
-    render() {
-        if (!Bowser.msie) {
```
Comment (sertony) [inline packages/components/src/UICompatibilityView/index.web.js:12]: @savelichalex FYI, this check was used to display a message for old versions of IE that we do not support their browser

File: packages/components/src/UICompatibilityView/UICompatibilityViewCommon.js [hunk 3]
```diff
@@ -1,34 +0,0 @@
...
-        justifyContent: 'center',
-        alignItems: 'center',
-    },
-});
-
-export default class UICompatibilityViewCommon extends UIComponent<null, null> {
-    renderNotCompatible() {
-        return (
-            <Text style={UITextStyle.primaryBodyRegular}>
-                {uiLocalized.WeAreSorryButYourBrowserVersionIsNotCompatible}
```
Comment (sertony) [inline packages/components/src/UICompatibilityView/UICompatibilityViewCommon.js:22]: If we don't need it anymore, you can clean up this translation as well

**Review by savelichalex — APPROVED** [review]

File: packages/components/src/UICompatibilityView/UICompatibilityViewCommon.js (hunk 3 above)
Comment (ghost) [reply packages/components/src/UICompatibilityView/UICompatibilityViewCommon.js:22]: @sertony done!

File: packages/components/src/UICompatibilityView/index.web.js (hunk 1 above)
Comment (ghost) [reply packages/components/src/UICompatibilityView/index.web.js:6]: @sertony done!

**Pull request merged**